### DISCUSSION TITLE
It's possible can i dump or view code asm or instruction of function?

### DISCUSSION THREAD

**Discussion by NRTnarathip** [vfsfitvnm/frida-il2cpp-bridge#565] [Q&A]
Hi, currently i have a dump.txt from Il2CPP.dump() 
but i want to know inner code assembly & instruction how function it's work

thanks🙏❤️.

**Comment by bluewave41** [comment]
Then you need to open your code with a disassembler. https://gist.github.com/BadMagic100/47096cbcf64ec0509cf75d48cfbdaea5
